src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/Admin.java

```diff
@@ -33,6 +37,114 @@ public void openSystemPreferences() {
     ElementHelper.clickOnElement(Helper.locateElementByTestId("system-prefs-accordion"));
   }
 
+  @Then("Click on \"Namespaces\" accordion")
```

Comment:
For each feature, separate class is maintained in cdapui e2e tests. 
To be in consistent with other code,  Can you please move namespaces related definitions to new class ?